src/LinearSolve.jl

```diff
@@ -38,14 +40,16 @@ function __init__()
   else
     IS_OPENBLAS[] = occursin("openblas", BLAS.get_config().loaded_libs[1].libname)
   end
+
+  @require Pardiso="46dd5b70-b6fb-5a00-ae2d-e8fea33afaf2" PardisoJL
 end
 
 export LUFactorization, SVDFactorization, QRFactorization, GenericFactorization,
        RFLUFactorizaation
-export KrylovJL, KrylovJL_CG, KrylovJL_GMRES, KrylovJL_BICGSTAB,
-       KrylovJL_MINRES,
+export KrylovJL, KrylovJL_CG, KrylovJL_GMRES, KrylovJL_BICGSTAB, KrylovJL_MINRES,
        IterativeSolversJL, IterativeSolversJL_CG, IterativeSolversJL_GMRES,
-       IterativeSolversJL_BICGSTAB, IterativeSolversJL_MINRES
+       IterativeSolversJL_BICGSTAB, IterativeSolversJL_MINRES,
+       PardisoJL
```

Comment:
the export will need to be in the requires statement

Reply:
done